plugins/auth.js

```diff
@@ -0,0 +1,3 @@
+export default ({ store }, inject) => {
```

Comment:
What for? Stores are always available. And there are Vuex store “getters” for easier access

Reply:
The plugin is just for a utility to make it shorter to access the auth stuff. What do you mean by Vuex store getters?